.github/workflows/build.yml

```diff
@@ -106,14 +106,17 @@ jobs:
     - name: Install the extension
       run: |
         set -eux
+        python -m pip install --pre jupyterlite-core
```

Comment:
Maybe there could be a pin on `jupyterlite-core` here, as new versions may bring some visual changes that would make the UI tests fail?

For example `jupyterlite-core==0.5.0` will be based on JupyterLab 4.3 and Notebook 7.3, which may have some CSS changes compared to `jupyterlite-core==0.4.x`.

Reply:
Yes, I'll do that.